### PR TITLE
Fix SickRage/sickrage-issues/issues/824

### PR DESCRIPTION
**Pull request by fernandog**

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/processTV.py [hunk 1]
```diff
@@ -200,6 +200,7 @@ def processDir(dirName, nzbName=None, process_method=None, force=False, is_prior
             # Unpack only if video file was not already extracted by 'postpone if no subs' feature
             rarContent = unRAR(path, rarFiles, force, result)
             files += rarContent
+            videoFiles = [x for x in fileList if helpers.isMediaFile(x)]
```
Comment (miigotu) [inline sickbeard/processTV.py]: videoFiles = [x for x in files if helpers.isMediaFile(x)]

**Comment by fernandog** [comment]
waiting user report it worked

**Comment by medariox** [comment]
One user reported this is indeed working. Should be merged to master ASAP.

**Comment by fernandog** [comment]
```
<SickRageUI|85418> Great! Thank you, medariox. It worked perfectly. :)
```

**Pull request merged**